### PR TITLE
added samsung smartthings plug

### PR DESCRIPTION
**Pull request by marc-gist**
All appears to work for the new Smartthings Plug, except the 'get' MQTT command.

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -1868,6 +1868,25 @@ const devices = [
...
         },
     },
+    {
+        zigbeeModel: ['outlet'],
+        model: 'IM6001-OTP05',
+        vendor: 'SmartThings',
+        description: 'Outlet',
+        supports: 'on/off',
+        fromZigbee: [fz.state, fz.ignore_onoff_report],
+        toZigbee: [tz.samsung_on_off],
```
Comment (Koenkk) [inline devices.js]: Does it also work when using `tz.on_off` instead of `tz.samsung_on_off`?

Comment (marc-gist) [reply devices.js]: no, crashes :(

Comment (marc-gist) [reply devices.js]: sorry, wrong spot, doesn't crash... nothing happens.  the changes i made seem to work.

Comment (Koenkk) [reply devices.js]: I would like to get a better understanding why this doesn't work. Could you post the log with `advanced -> log_level: debug` with both converters when turning on/off the switch?

Comment (marc-gist) [reply devices.js]: you want me to change code to
`toZigbee: [tz.samsung_on_off, tz.on_off]]` 
and log... or just
`toZigbee: [tz.on_off]]`

Comment (Koenkk) [reply devices.js]: just `toZigbee: [tz.on_off]`

Comment (marc-gist) [reply devices.js]: strange... working now... i swear it didn't first time... perhaps a timing/reload issue (didn't save file properly before reloading).

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**